pom.xml

```diff
@@ -20,16 +20,37 @@
 		<dcsa.version>0.8.34</dcsa.version>
```

Comment:
Isn't there a "<java.version>11</java.version>" hiding somewhere around here that should also be removed?